tests/test_assignment.py

```diff
@@ -5,7 +5,19 @@
 import autograder.assignment
 import autograder.util.invoke
 
+BASE_ERROR_MESSAGE = "Incorrect result."
+SKIPPING_QUESTION_MESSAGE = "Grading stopped, skipping question..."
+
 class TestAssignment(unittest.TestCase):
+    class Q0(autograder.question.Question):
```

Comment:
I would start calling the question classes something more useful like "QuestionHardFail".